hello-world.py

```diff
@@ -3,4 +3,6 @@ def hello_world():
   Implement this function so it displays 'Hello World' to the screen when called
   """
   
+  print("Howdy World!")
```

Comment:
Hi @elliotwhitehead-student great work on attempting assignment. You may want to read the assignments specs again. Make a new PR and we'll take a look afterwards.

Reply:
Thanks! All fixed.